<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtCustomCommandConfiguration</class>
 <widget class="QDialog" name="LXQtCustomCommandConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Command Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="repeatCheckBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Repeat command after:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="repeatTimerSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> second(s)</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>86400</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="iconBrowseButton">
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="maxWidthLabel">
          <property name="text">
           <string>Max Width</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="fontLabel">
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="5">
         <widget class="QLabel" name="commandLabel">
          <property name="text">
           <string>Command</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="iconLabel">
          <property name="text">
           <string>Icon</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QCheckBox" name="autoRotateCheckBox">
          <property name="text">
           <string>Autorotate when the panel is vertical</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QSpinBox" name="maxWidthSpinBox">
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="iconLineEdit">
          <property name="placeholderText">
           <string>Use icon from theme or browse file</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QCheckBox" name="runWithBashCheckBox">
          <property name="text">
           <string>Run with &quot;bash -c&quot;</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QPushButton" name="fontButton">
          <property name="text">
           <string>Select Font</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QLineEdit" name="textLineEdit">
          <property name="text">
           <string>%1</string>
          </property>
          <property name="placeholderText">
           <string>Use %1 to display command output</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="textLabel">
          <property name="text">
           <string>Text</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="commandPlainTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="plainText">
           <string>echo Configure...</string>
          </property>
          <property name="placeholderText">
           <string>Command to run</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QCheckBox" name="outputImageCheckBox">
          <property name="text">
           <string>Command outputs an image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mouse Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="clickLabel">
        <property name="text">
         <string>Click</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="wheelUpLineEdit">
        <property name="placeholderText">
         <string>Command to run</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="wheelDownLineEdit">
        <property name="placeholderText">
         <string>Command to run</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="clickLineEdit">
        <property name="placeholderText">
         <string>Command to run</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="wheelUpLabel">
        <property name="text">
         <string>Wheel Up</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="wheelDownLabel">
        <property name="text">
         <string>Wheel Down</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>autoRotateCheckBox</tabstop>
  <tabstop>fontButton</tabstop>
  <tabstop>commandPlainTextEdit</tabstop>
  <tabstop>runWithBashCheckBox</tabstop>
  <tabstop>repeatCheckBox</tabstop>
  <tabstop>repeatTimerSpinBox</tabstop>
  <tabstop>iconLineEdit</tabstop>
  <tabstop>iconBrowseButton</tabstop>
  <tabstop>textLineEdit</tabstop>
  <tabstop>maxWidthSpinBox</tabstop>
  <tabstop>clickLineEdit</tabstop>
  <tabstop>wheelUpLineEdit</tabstop>
  <tabstop>wheelDownLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LXQtCustomCommandConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LXQtCustomCommandConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
